runtime/doc/helphelp.txt

```diff
@@ -433,17 +433,31 @@ To quote a block of ex-commands verbatim, place a greater than (>) character
 at the end of the line before the block and a less than (<) character as the
 first non-blank on a line following the block.  Any line starting in column 1
 also implicitly stops the block of ex-commands before it.  E.g. >
-    function Example_Func()
-	echo "Example"
-    endfunction
+	function Example_Func()
+	  echo "Example"
+	endfunction
 <
-It's possible to add Vim syntax highlighting support to code examples.  This
-can be done by adding "vim" after the greater than (>) character (">vim").
-E.g: >vim
-    function Example_Func()
-	echo "Example"
-    endfunction
+
+To add annotation in the block, place the annotation (ex: "lua") after a
+greater than (>) character.  E.g: >lua
+	print("hello")
+<
+Note: uses lua syntax highlighting, if "lua" is in |g:help_example_languages|.
+It's possible to add Vim syntax highlighting support to code
+examples.  E.g: >vim
+	function Example_Func()
+	  echo "Example"
+	endfunction
 <
+						 *g:help_example_languages*
+If you want to change the syntax highlighting in the block, you can
+change it like this: >
+	:let g:help_example_languages = #{ vim: 'vim', sh: 'bash' }
+The key is the annotation marker name and the value is the 'syntax' name.
+When not configured, help files support Vim script highlighting only.
+Note: When setting "g:help_example_languages", if you do not add "vim", Vim
+When setting "g:help_example_languages" to an empty value, syntax highlighting for embedded languages will be disabled.
```

Comment:
This was posted to vim_dev, but it doesn't seem to be reflected on GitHub, so I'm rewriting it.

- The flow from the previous line seems strange.
- Exceeds 80 columns.

Reply:
> Exceeds 80 columns.

Thanks.

Reply:
> The flow from the previous line seems strange.

I have updated the documentation.